src/frontend/src/utils/utils.ts

```diff
@@ -1,3 +1,8 @@
+// Ensures the argument has a given type (from ts' perspective, not js)
+export function assertType<T>(_: T) {
```

Comment:
@peterpeterparker do you know if there's a ts builtin that could be used here?

Reply:
That's the assertion function we are using:

```
export const assertNonNullish: <T>(
  value: T,
  message?: string
) => asserts value is NonNullable<T> = <T>(
  value: T,
  message?: string
): void => {
  if (value === null || value === undefined) {
    throw new NullishError(message);
  }
};
```

`asserts` is the TypeScript keyword.

https://github.com/dfinity/ic-js/blob/main/packages/utils/src/utils/asserts.utils.ts

It throws an error.

Reply:
I know that one, I've read the blog post 😉 

`assertTy` does something different, in that it doesn't care about the JS value at all, but only serves to assert the TypeScript type